tests/PHPStan/Rules/Comparison/MatchExpressionRuleTest.php

```diff
@@ -153,7 +153,11 @@ public function testEnums(): void
 				56,
 			],
 			[
-				'Match arm comparison between *NEVER* and MatchEnums\Foo is always false.',
+				'Match arm comparison between MatchEnums\Foo::THREE and MatchEnums\Foo::THREE is always true.',
```

Comment:
this new error is a bit misleading. not sure we want to keep it like that.

Reply:
It's fine :)